collie/model/base/base_pipeline.py

```diff
@@ -782,6 +782,13 @@ def item_item_similarity(self, item_id: int) -> pd.Series:
         always be the item itself.
 
         """
+        if item_id > self.hparams.num_items:
```

Comment:
Shoot, does this need to be a `>=`. I think `self.hparams.num_items` will still lead to an `IndexError`. Can you test this and confirm? If it does, then can you change all four spots to `>=`.

Reply:
oh right! let me confirm.

Reply:
You are correct! Great catch! So, in the tests do you prefer:
```
nonexistent_item_id = train.num_items + 1
```
or
```
nonexistent_item_id = train.num_items 
```
?

Reply:
I think I prefer the latter for the tests.